packages/nimble-components/src/utilities/tests/rich-text-markdown-string.ts

```diff
@@ -1 +1 @@
-export const richTextMarkdownString = 'Supported rich text formatting options:\n1. **Bold**\n2. *Italics*\n3. Numbered lists\n   1. Option 1\n   2. Option 2\n4. Bulleted lists\n   * Option 1\n   * Option 2\n5. Absolute link: <https://nimble.ni.dev/>\n 6. @mention:\n    1. User pattern: <user:1>\n    2. HTTPS pattern: <https://user/2>';
+export const richTextMarkdownString = 'Supported rich text formatting options:\n1. **Bold**\n2. *Italics*\n3. Numbered lists\n   1. Option 1\n   2. Option 2\n4. Bulleted lists\n   * Option 1\n   * Option 2\n5. Absolute link: ***<https://nimble.ni.dev/>***\n 6. @mention:\n    1. User pattern: <user:1>\n    2. HTTPS pattern: <https://user/2>';
```

Comment:
I don't think this is a good way to test this change. This test string should render each construct in its default configuration since it's used by user-facing examples and by tests that cover behavior of common cases.

Instead we should have some dedicated matrix tests that ensure absolute links and @ mentions render correctly when configured to be bold, italics, and both.